drivers/gles3/shaders/scene.glsl

```diff
@@ -1521,6 +1521,7 @@ void main() {
 	if (alpha < alpha_scissor_threshold) {
 		discard;
 	}
+	alpha = 1.0;
```

Comment:
This fixes a bug that became more obvious as a result of enabling post effects. 

This code matches the behaviour of the RD renderers:
https://github.com/godotengine/godot/blob/11d3768132582d192b8464769f26b493ae822321/servers/rendering/renderer_rd/shaders/forward_clustered/scene_forward_clustered.glsl#L1066-L1068